tuning/trackers/filelogging_tracker.py

```diff
@@ -80,7 +80,10 @@ def __init__(self, tracker_config: FileLoggingTrackerConfig):
                 which contains the location of file where logs are recorded.
         """
         super().__init__(name="file_logger", tracker_config=tracker_config)
-        self.logger = logging.get_logger("file_logging_tracker")
+        # Configure log level
+        LOGLEVEL = os.environ.get("LOG_LEVEL", "WARNING").upper()
+        logging.basicConfig(level=LOGLEVEL)
```

Comment:
It'd be helpful also if in each file, you could do:
```
logger = logging.getLogger(__name__)
```
This will specify in the logs which module the log came from, example:
```
INFO:__main__:Started
INFO:mylib:Doing something
INFO:__main__:Finished
```

Reply:
Yea that's a good idea. I will do that.